### ISSUE TITLE
Finding ancestor by custom tag_name doesn't work

### ISSUE THREAD

**Issue by abotalov** [watir/nerodia#13]
## Meta -
Nerodia Version: 0.9.0

## Expected Behavior -
Finding ancestor by custom tag_name works.

## Actual Behavior -
Finding ancestor by custom tag_name doesn't work.

## Steps to reproduce -
```html
<parent-element>
  <child-element>text</child-element>
</parent-element>
```
```python
br = nerodia.browser.Browser()
br.goto("https://fiddle.jshell.net/a4gwk3by/show/")
a = br.iframe().element(tag_name="child-element").parent(tag_name="parent-element").text
```
Raises `TypeError: 'NoneType' object is not callable`

**Comment by lmtierney** [comment]
This bug is also present in `Watir`

**Comment by lmtierney** [comment]
Fixed in ebbc8876fee427f3ee9116ea9c6014f80be9055f, will be in the next release
